### DISCUSSION TITLE
Add social icons in custom page

### DISCUSSION THREAD

**Discussion by shanalikhan** [alshedivat/al-folio#2924] [Q&A] [answered]
Hi, 
I've created page with this metadata
```
---
layout: page
title: Healthcare Contributions
permalink: /healthcare
subtitle: .
img: assets/img/12.jpg
social: true
---
```

But still social icons are not appearing like in about us page, how to enable in custom pages?

**Comment by george-gca** [comment, marked as answer]
Currently only the about page supports social icons. You have 2 options:

1. Enable social icons in the header:
https://github.com/alshedivat/al-folio/blob/807629dd2f45044f0f5f2f66a71445387e3e2693/_config.yml#L400
2. Create a new layout in `_layouts/` similar to the page that you want, but adding the socials part from `_layouts/about.liquid`.

**Comment by tropikalkimani** [comment]
Have you tried embedding the icon's code on the html code ?
